### PR TITLE
Update test for 'noopener' feature of window.open

### PR DESCRIPTION
**Pull request by cdumez**
Update test for 'noopener' feature of window.open to reflect the changes made in
https://github.com/whatwg/html/pull/4079.

### CODE REVIEW

**Comment by cdumez** [comment]
cc @zcorpan

File: html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html [hunk 1]
```diff
@@ -114,20 +116,29 @@
...
     var win = window.open(windowURL, '', feature);
     assert_equals(win, null, `"${feature}" should activate feature "noopener"`);
   });
-}, '"noopener" should be based on name (key), not value');
+}, 'Integer values other than 0 should activate the feature');
+
+test (t => {
+  var featureVariants = [
+    'noopener=0',
+    'noopener=0.5',
```
Comment (zcorpan) [inline html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html]: Can you add `noopener=error` here?

**Review by zcorpan — APPROVED** [review]
LGTM % 1 comment

File: html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html (hunk 1 above)
Comment (cdumez) [reply html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-noopener.html]: Done.

**Pull request merged**